### PR TITLE
Feature/responsiveness

### PR DESCRIPTION
**Pull request by m1073496**
## What does this PR do?

- Fixes responsiveness for small mobile devices as well as ipad/tablet devices
- Creates a mediaQueries file and import into base file
- Adjust image sizing for different screen sizes
- Adds hover effect to columns
- Adds aria labels to form --> accessibility audit on lighthouse at 100%
- Adds tab indexes to important fields
- Creates H2s for Hydration column and Sleep column

## How should it be tested in the UI?

- Take a look-see to check ^^
- Run lighthouse audit for accessibility
- Use screen reader to find issues with readability

### CODE REVIEW

**Comment by m1073496** [comment]
[Modify App Design](https://trello.com/c/gbmhIxn9/8-modify-app-design)

File: src/css/_activityColumn.scss [hunk 1]
```diff
@@ -8,6 +8,10 @@
   align-items: center;
   text-align: center;
   justify-content: space-around;
+
```
Comment (Shakikka) [inline src/css/_activityColumn.scss]: These hover states crack me up.

File: src/css/_mediaQueries.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    grid-column: 1 / 2;
+    grid-row: 5 / 6;
+  }
+
+  .sidebar-header-challengeImage {
+    width: 40%;
+  }
+}
+
+@media screen and (max-width: 800px) and (min-width: 700px) {
```
Comment (Shakikka) [inline src/css/_mediaQueries.scss]: It is doing a weird thing where it is going to 2 columns, then back to 3 columns, before going back to 2 columns as the screen gets smaller. Perhaps try out something like `(max-width: 1024px) and (min-width: 550px)`

**Review by Shakikka — CHANGES_REQUESTED** [review]
Overall lookin good, but I think these media query breakpoints need a tweak.

File: src/css/_mediaQueries.scss (hunk 2 above)
Comment (m1073496) [reply src/css/_mediaQueries.scss]: got it! thanks!

File: src/css/_activityColumn.scss (hunk 1 above)
Comment (m1073496) [reply src/css/_activityColumn.scss]: removed 😂

**Review by Shakikka — APPROVED** [review]

**Review by Shakikka — APPROVED** [review]

**Pull request merged**